css/reset.css

```diff
@@ -1,3 +1,8 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
 html, body {
     margin: 0;
     padding: 0;
```

Comment:
you can remove this as you're using * I guess

Reply:
Thanks for pointing that out!